src/mono/wasm/runtime/roots.ts

```diff
@@ -217,7 +217,7 @@ export class WasmRootBufferImpl implements WasmRootBuffer {
     get(index: number): ManagedPointer {
         this._check_in_range(index);
         const offset = this.get_address_32(index);
-        return <any>Module.HEAPU32[offset];
+        return <any>localHeapViewU32()[offset];
```

Comment:
roots and root buffers don't need growth checks; we pre-allocated them so as long as we do a growth check after the initial allocation the addresses will always be visible.

Reply:
I put back `Module.HEAPU32` to all root code, correct also for MT ?

Reply:
I'm not sure. I think `grow` would disconnect the views and they will be returning `undefined`

Reply:
Good point